src/nessi.jl

```diff
@@ -47,15 +58,75 @@ function nessi_read_tv(h5g::HDF5.Group)
   return tv
 end
 
-function nessi_read_mat(h5g::HDF5.Group)
+function _nessi_read_mat(h5g::HDF5.Group)
   mat = read(h5g, "mat")
   return mat
 end
 
-function nessi_read_gf(h5g::HDF5.Group)
-  les = nessi_read_les(h5g)
-  ret = nessi_read_ret(h5g)
-  tv  = nessi_read_tv(h5g)
-  mat = nessi_read_mat(h5g)
-  return (;les, ret, tv, mat)
+function NessiGFData(h5g::HDF5.Group)
```

Comment:
@HugoStrand Hugo, would you mind checking this function? You are apparently the best person for this task.